packages/block-editor/src/components/media-placeholder/index.native.js

```diff
@@ -15,10 +15,10 @@ import { MediaUpload, MEDIA_TYPE_IMAGE, MEDIA_TYPE_VIDEO } from '@wordpress/bloc
 import styles from './styles.scss';
 
 function MediaPlaceholder( props ) {
-	const { mediaType, labels = {}, icon, onSelectURL } = props;
+	const { allowedTypes = [], labels = {}, icon, onSelect } = props;
 
-	const isImage = MEDIA_TYPE_IMAGE === mediaType;
-	const isVideo = MEDIA_TYPE_VIDEO === mediaType;
+	const isImage = allowedTypes.includes( MEDIA_TYPE_IMAGE );
+	const isVideo = allowedTypes.includes( MEDIA_TYPE_VIDEO );
```

Comment:
Sorry, I just realized it's a bit more complicated than this. The web code was also checking just the first item (e.g. `'image' === allowedTypes[ 0 ]`), but only because it was also checking that there was only one item `1 === allowedTypes.length`.

I guess we took a shortcut here since this was only used with one type on mobile but, for instance, the `media-text` block uses a placeholder that allows `[ 'image', 'video' ]`. 

We only want to check `isImage`/`isVideo` if it's *only* one of those, because this is all to control the copy around the placeholder. So if it accepts more than one media type we use a more generic message.

I see that we keep those generic "Media" labels (except for instructions), so the logic should be similar to the web implementation, with just the different strings for each placeholder type.

Reply:
Hmm, I think you're right and we have to use the similar implementation in `native.js` where `Media` label as you said is the default one for more than one type.

Reply:
One more question related to `allowedTypes` - what should be a `default` icon in [media-upload](https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/media-upload/index.native.js#L46-#L54) for case when there are more than one type?

Reply:
I think the one media & text block uses on web is suitable. 

<img width="569" alt="Screen Shot 2019-08-23 at 15 25 34" src="https://user-images.githubusercontent.com/5032900/63592491-60322a00-c5ba-11e9-91de-1eabb2f5868c.png">

Reply:
@pinarol 	, sorry for the misunderstanding but unfortunately I'm looking for the icon for android bottom sheet and cannot find the proper one.
![image](https://user-images.githubusercontent.com/22746080/63596231-0bdb7a00-c5bb-11e9-8b3c-ab1fa9ec251f.png)

Reply:
ah sorry gotcha. I added this comment about it https://github.com/WordPress/gutenberg/pull/17145/files#r317193740

I wonder what @koke thinks though, let's hear from him also.

Reply:
Why do we need a default icon? From what I can see the icon is passed as a prop. The placeholder doesn't have any logic so far to pick an icon depending on allowed types

Reply:
An `icon` is passed to the `media-placeholder`. The screenshot above comes from a bottom sheet on Android platform